<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Recent Commit Activity</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-d1b12f2568ecbe55642fee6aa00bd082.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="style.css">
</head>

<body class="nav-fixed fullcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">PublicMicrobiomeProjectHub</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./overview.html"> 
<span class="menu-text">Overview</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./commit_log.html" aria-current="page"> 
<span class="menu-text">CommitLog</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Recent Commit Activity</h1>
<p class="subtitle lead">This site automatically tracks a curated list of GitHub projects funded by NIH grant <a href="https://reporter.nih.gov/project-details/9963159">R01CA230551</a> from the National Cancer Institute to Levi Waldron at the CUNY Graduate School of Public Health and Health Policy</p>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="waldronlabbugsigdbexports" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbexports"><a href="https://github.com/waldronlab/BugSigDBExports/commit/573f86337ecbe2ae0fc8a5a2e34286790846848f"><code>waldronlab/BugSigDBExports</code></a></h3>
<p><strong>Message:</strong> Hourly export update</p>
<p><strong>Author:</strong> GitHub Actions Bot</p>
<p><strong>Date:</strong> 2025-08-11 14:08:21 UTC</p>
<hr>
</section>
<section id="bd4everyonemicrobiomeproject" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyonemicrobiomeproject"><a href="https://github.com/bd4everyone/MicrobiomeProject/commit/828ca0a1525a661249f3e53501f21f77f9a5be8c"><code>bd4everyone/MicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> Update index.qmd</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-08-11 13:56:16 UTC</p>
<hr>
</section>
<section id="bd4everyonemicrobiomeproject-1" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyonemicrobiomeproject-1"><a href="https://github.com/bd4everyone/MicrobiomeProject/commit/e88c3a6f21a677e197cbc75f3c82127a5b6e3847"><code>bd4everyone/MicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> Update _quarto.yml</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-08-11 13:47:07 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbexports-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbexports-1"><a href="https://github.com/waldronlab/BugSigDBExports/commit/7db83ff2b8960d7470a454d306bcd899fe43412d"><code>waldronlab/BugSigDBExports</code></a></h3>
<p><strong>Message:</strong> Hourly export update</p>
<p><strong>Author:</strong> GitHub Actions Bot</p>
<p><strong>Date:</strong> 2025-08-11 13:18:46 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbexports-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbexports-2"><a href="https://github.com/waldronlab/BugSigDBExports/commit/f1d38453615743dd8052239222d9058df50cb9ed"><code>waldronlab/BugSigDBExports</code></a></h3>
<p><strong>Message:</strong> Hourly export update</p>
<p><strong>Author:</strong> GitHub Actions Bot</p>
<p><strong>Date:</strong> 2025-08-11 12:06:01 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbexports-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbexports-3"><a href="https://github.com/waldronlab/BugSigDBExports/commit/f9088bb81b5abd52587b7274b2450b41dec9e137"><code>waldronlab/BugSigDBExports</code></a></h3>
<p><strong>Message:</strong> Hourly export update</p>
<p><strong>Author:</strong> GitHub Actions Bot</p>
<p><strong>Date:</strong> 2025-08-11 11:06:21 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbexports-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbexports-4"><a href="https://github.com/waldronlab/BugSigDBExports/commit/e959db58795b6ed7b6f9ebb1bd71d176b237937a"><code>waldronlab/BugSigDBExports</code></a></h3>
<p><strong>Message:</strong> Hourly export update</p>
<p><strong>Author:</strong> GitHub Actions Bot</p>
<p><strong>Date:</strong> 2025-08-11 10:06:13 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzexports" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzexports"><a href="https://github.com/waldronlab/bugphyzzExports/commit/0b2bae419f86cc8a1d9933600f7bf7233d22d723"><code>waldronlab/bugphyzzExports</code></a></h3>
<p><strong>Message:</strong> Update exports</p>
<p><strong>Author:</strong> github-actions</p>
<p><strong>Date:</strong> 2025-08-10 23:41:52 UTC</p>
<hr>
</section>
<section id="bd4everyonemicrobiomeproject-2" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyonemicrobiomeproject-2"><a href="https://github.com/bd4everyone/MicrobiomeProject/commit/d0027832f25c0f1efc3b17e2bbe5bdb0fc323879"><code>bd4everyone/MicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> Delete projects.qmd</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-08-07 17:05:52 UTC</p>
<hr>
</section>
<section id="bd4everyonemicrobiomeproject-3" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyonemicrobiomeproject-3"><a href="https://github.com/bd4everyone/MicrobiomeProject/commit/18b1be19e8675dee2c3af7fb6010fbfc8838ff42"><code>bd4everyone/MicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> Delete commit_log.qmd</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-08-07 17:05:14 UTC</p>
<hr>
</section>
<section id="bd4everyonemicrobiomeproject-4" class="level3">
<h3 class="anchored" data-anchor-id="bd4everyonemicrobiomeproject-4"><a href="https://github.com/bd4everyone/MicrobiomeProject/commit/2446f09370b1a231964c89bd02624ea26a738c46"><code>bd4everyone/MicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> Add files via upload</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-08-07 17:03:44 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbstats" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbstats"><a href="https://github.com/waldronlab/BugSigDBStats/commit/42e820bd3a1ab14974273ab9f963da4998354b55"><code>waldronlab/BugSigDBStats</code></a></h3>
<p><strong>Message:</strong> Merge pull request #19 from annemarie-sharp/improve-rmd-tables</p>
<p><strong>Author:</strong> Chloe Mirzayi</p>
<p><strong>Date:</strong> 2025-08-06 16:20:41 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzexports-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzexports-1"><a href="https://github.com/waldronlab/bugphyzzExports/commit/34219d3ec8847790fff2b95f21dabd98efeefe15"><code>waldronlab/bugphyzzExports</code></a></h3>
<p><strong>Message:</strong> Update exports</p>
<p><strong>Author:</strong> github-actions</p>
<p><strong>Date:</strong> 2025-08-03 23:44:06 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbstats-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbstats-1"><a href="https://github.com/waldronlab/BugSigDBStats/commit/bf75e0a8e844b2e1dc10ba9bc7a779a56ffacc0b"><code>waldronlab/BugSigDBStats</code></a></h3>
<p><strong>Message:</strong> Create nicer-looking tables in Rmd; update .gitignore and .Rbuildignore</p>
<p><strong>Author:</strong> Anne-Marie Sharp</p>
<p><strong>Date:</strong> 2025-08-01 19:46:26 UTC</p>
<hr>
</section>
<section id="waldronlabpublicmicrobiomeproject" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicmicrobiomeproject"><a href="https://github.com/waldronlab/PublicMicrobiomeProject/commit/1885490acb0ea4a41b7f96e22b1257bcbfd6e7c4"><code>waldronlab/PublicMicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> Link &amp; icon (#2)</p>
<p><strong>Author:</strong> bd4everyone</p>
<p><strong>Date:</strong> 2025-07-31 14:26:32 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataanalyses" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataanalyses"><a href="https://github.com/waldronlab/curatedMetagenomicDataAnalyses/commit/7b9a72dc470de4714f83c141cadccabe49762f9d"><code>waldronlab/curatedMetagenomicDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Minor comment in setup stage</p>
<p><strong>Author:</strong> Giacomo Antonello</p>
<p><strong>Date:</strong> 2025-07-30 14:43:10 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataanalyses-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataanalyses-1"><a href="https://github.com/waldronlab/curatedMetagenomicDataAnalyses/commit/ae312cbc794c851e5f69bf77a05a8ca72fe93ca1"><code>waldronlab/curatedMetagenomicDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Update README.md with Zenodo badge</p>
<p><strong>Author:</strong> Giacomo Antonello</p>
<p><strong>Date:</strong> 2025-07-30 14:42:10 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataanalyses-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataanalyses-2"><a href="https://github.com/waldronlab/curatedMetagenomicDataAnalyses/commit/089592d5b136c615ad2f1b4a9148c3144ee4e114"><code>waldronlab/curatedMetagenomicDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Major streamlining of git repository after paper re-submisison. Data are now on Zenodo, see https://doi.org/10.5281/zenodo.15856813</p>
<p><strong>Author:</strong> Giacomo Antonello</p>
<p><strong>Date:</strong> 2025-07-30 14:37:42 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataanalyses-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataanalyses-3"><a href="https://github.com/waldronlab/curatedMetagenomicDataAnalyses/commit/82d7e377a4cc5ccf0b35f5469a4821e4d4f17e08"><code>waldronlab/curatedMetagenomicDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘main’ of github.com:waldronlab/curatedMetagenomicDataAnalyses</p>
<p><strong>Author:</strong> g-antonello</p>
<p><strong>Date:</strong> 2025-07-30 09:53:31 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataanalyses-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataanalyses-4"><a href="https://github.com/waldronlab/curatedMetagenomicDataAnalyses/commit/0f41bde0fc745b8a79c3857905a9f7595d9cc3c1"><code>waldronlab/curatedMetagenomicDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Added pre-set conda environment for quick install</p>
<p><strong>Author:</strong> g-antonello</p>
<p><strong>Date:</strong> 2025-07-30 09:52:46 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/37df7650985db64f0e06b72728dab7909a391ce8"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> handling updated delimiter</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-28 18:48:32 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzexports-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzexports-2"><a href="https://github.com/waldronlab/bugphyzzExports/commit/e1a6f3923c82739e03fad6e0d9f0abfbf8817029"><code>waldronlab/bugphyzzExports</code></a></h3>
<p><strong>Message:</strong> Update exports</p>
<p><strong>Author:</strong> github-actions</p>
<p><strong>Date:</strong> 2025-07-27 23:44:33 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-1" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-1"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/ff26cc4cee4d37748f626a21c69a90aad484b624"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> update delimiter</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-25 22:53:37 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatacuration" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatacuration"><a href="https://github.com/waldronlab/curatedMetagenomicDataCuration/commit/5da05843c99a9e580d4271ef5a120d78332ef0b0"><code>waldronlab/curatedMetagenomicDataCuration</code></a></h3>
<p><strong>Message:</strong> Merge pull request #102 from waldronlab/issue/99</p>
<p><strong>Author:</strong> jwokaty</p>
<p><strong>Date:</strong> 2025-07-22 19:53:52 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatacuration-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatacuration-1"><a href="https://github.com/waldronlab/curatedMetagenomicDataCuration/commit/c69cb5e0778efddf3ea31b7278ff8fe4a4044c7e"><code>waldronlab/curatedMetagenomicDataCuration</code></a></h3>
<p><strong>Message:</strong> Get last name</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-07-22 19:32:29 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatacuration-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatacuration-2"><a href="https://github.com/waldronlab/curatedMetagenomicDataCuration/commit/9989204daeda4bd04d0bdcd5abb15f51b4041a09"><code>waldronlab/curatedMetagenomicDataCuration</code></a></h3>
<p><strong>Message:</strong> Push to default branch</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-07-22 19:08:17 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdata"><a href="https://github.com/waldronlab/curatedMetagenomicData/commit/e5b5f83fb46a28e61a25546bf0d70c2bc0e39d4d"><code>waldronlab/curatedMetagenomicData</code></a></h3>
<p><strong>Message:</strong> Merge pull request #319 from shbrief/master</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2025-07-18 07:53:05 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatacuration-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatacuration-3"><a href="https://github.com/waldronlab/curatedMetagenomicDataCuration/commit/987fefd09142e94dbf081e4b38020dd0bac84cba"><code>waldronlab/curatedMetagenomicDataCuration</code></a></h3>
<p><strong>Message:</strong> Update cMD_doi.yml</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-16 18:36:13 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatacuration-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatacuration-4"><a href="https://github.com/waldronlab/curatedMetagenomicDataCuration/commit/c98c5708f56298f1dd09a5d4be9b484fbdf31e4d"><code>waldronlab/curatedMetagenomicDataCuration</code></a></h3>
<p><strong>Message:</strong> Make BioProject as a required input for new study template</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-07-14 17:09:11 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzexports-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzexports-3"><a href="https://github.com/waldronlab/bugphyzzExports/commit/b83734496bb5a71ecd56edc82ffb450fc2a0ced1"><code>waldronlab/bugphyzzExports</code></a></h3>
<p><strong>Message:</strong> Update exports</p>
<p><strong>Author:</strong> github-actions</p>
<p><strong>Date:</strong> 2025-07-13 23:43:29 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigsimple" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigsimple"><a href="https://github.com/waldronlab/bugSigSimple/commit/028352f0da94ae1a6315ad2ea189e1ab365b8987"><code>waldronlab/bugSigSimple</code></a></h3>
<p><strong>Message:</strong> Merge pull request #31 from annemarie-sharp/unittests-bugSigSimple</p>
<p><strong>Author:</strong> Chloe Mirzayi</p>
<p><strong>Date:</strong> 2025-07-10 21:46:13 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigsimple-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigsimple-1"><a href="https://github.com/waldronlab/bugSigSimple/commit/25a0e563b997f5d040c86a21418c71651ae73356"><code>waldronlab/bugSigSimple</code></a></h3>
<p><strong>Message:</strong> Add unit tests in testthat folder</p>
<p><strong>Author:</strong> Anne-Marie Sharp</p>
<p><strong>Date:</strong> 2025-07-10 15:08:13 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdb" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdb"><a href="https://github.com/waldronlab/BugSigDB/commit/73b9399298f3f4f87f43f5a54ddc00890b042edf"><code>waldronlab/BugSigDB</code></a></h3>
<p><strong>Message:</strong> Update cron image: ghcr.io/wikiteq/cron:20250709-2da693f (#272)</p>
<p><strong>Author:</strong> Pavel Astakhov</p>
<p><strong>Date:</strong> 2025-07-09 14:48:30 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdb-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdb-1"><a href="https://github.com/waldronlab/BugSigDB/commit/9d24c6b300d14ad2f7dae852ad34d515de2c7704"><code>waldronlab/BugSigDB</code></a></h3>
<p><strong>Message:</strong> Set –binlog_expire_logs_seconds=86400 (#271)</p>
<p><strong>Author:</strong> Pavel Astakhov</p>
<p><strong>Date:</strong> 2025-07-09 13:50:23 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzexports-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzexports-4"><a href="https://github.com/waldronlab/bugphyzzExports/commit/787cf748d03fd5d43b3cae8b7ce17022a5bfd5c1"><code>waldronlab/bugphyzzExports</code></a></h3>
<p><strong>Message:</strong> Update exports</p>
<p><strong>Author:</strong> github-actions</p>
<p><strong>Date:</strong> 2025-07-06 23:43:08 UTC</p>
<hr>
</section>
<section id="waldronlabpublicmicrobiomeproject-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicmicrobiomeproject-1"><a href="https://github.com/waldronlab/PublicMicrobiomeProject/commit/ee242416315817fb8f32ae1acad43a0afdfae4b9"><code>waldronlab/PublicMicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> initial commit</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2025-07-06 21:03:02 UTC</p>
<hr>
</section>
<section id="waldronlabpublicmicrobiomeproject-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabpublicmicrobiomeproject-2"><a href="https://github.com/waldronlab/PublicMicrobiomeProject/commit/1b1a580ef4927393d0e83c157ccd1ee3f7873548"><code>waldronlab/PublicMicrobiomeProject</code></a></h3>
<p><strong>Message:</strong> Initial commit</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2025-07-06 21:01:09 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdb-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdb-2"><a href="https://github.com/waldronlab/BugSigDB/commit/f1e9ea3365ce05319fd7f4285aef7aeef5794eeb"><code>waldronlab/BugSigDB</code></a></h3>
<p><strong>Message:</strong> Add mediawiki-ddos to varnish label (#270)</p>
<p><strong>Author:</strong> Pavel Astakhov</p>
<p><strong>Date:</strong> 2025-07-04 11:58:40 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdb-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdb-3"><a href="https://github.com/waldronlab/BugSigDB/commit/5448ecb41bf4337a7042659de0f68894a98a0816"><code>waldronlab/BugSigDB</code></a></h3>
<p><strong>Message:</strong> Update cron container (cron:20250704-10) (#269)</p>
<p><strong>Author:</strong> Pavel Astakhov</p>
<p><strong>Date:</strong> 2025-07-04 11:30:12 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdb-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdb-4"><a href="https://github.com/waldronlab/BugSigDB/commit/8db4fe026d47ab5856ee5c135ecc8bf6bef7c938"><code>waldronlab/BugSigDB</code></a></h3>
<p><strong>Message:</strong> Set PHP_MEMORY_LIMIT=2G to the matomo container (#267)</p>
<p><strong>Author:</strong> Pavel Astakhov</p>
<p><strong>Date:</strong> 2025-07-01 07:14:02 UTC</p>
<hr>
</section>
<section id="waldronlabtaxppro" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxppro"><a href="https://github.com/waldronlab/taxPPro/commit/c6d1fa560976b2475191d149d249a1dc0c9597c0"><code>waldronlab/taxPPro</code></a></h3>
<p><strong>Message:</strong> Add taxizedb, bump version to 1.0.1</p>
<p><strong>Author:</strong> A Wokaty</p>
<p><strong>Date:</strong> 2025-06-30 00:00:45 UTC</p>
<hr>
</section>
<section id="seandavicuratedmetagenomicsnextflow" class="level3">
<h3 class="anchored" data-anchor-id="seandavicuratedmetagenomicsnextflow"><a href="https://github.com/seandavi/curatedMetagenomicsNextflow/commit/4cde6fbcc090b67be78ba60e26330880a8d9c8eb"><code>seandavi/curatedMetagenomicsNextflow</code></a></h3>
<p><strong>Message:</strong> refactor pkl file munging</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-06-25 20:46:59 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdata-1"><a href="https://github.com/waldronlab/curatedMetagenomicData/commit/f383cef193325e3647de59b6477d1ac58570c653"><code>waldronlab/curatedMetagenomicData</code></a></h3>
<p><strong>Message:</strong> Updates following the reviewer’s suggestions</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-06-06 03:31:59 UTC</p>
<hr>
</section>
<section id="seandavicuratedmetagenomicsnextflow-1" class="level3">
<h3 class="anchored" data-anchor-id="seandavicuratedmetagenomicsnextflow-1"><a href="https://github.com/seandavi/curatedMetagenomicsNextflow/commit/4bd6007ef767063385d999862c8507498161c67b"><code>seandavi/curatedMetagenomicsNextflow</code></a></h3>
<p><strong>Message:</strong> aws cli no longer a dependency</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-06-03 15:34:31 UTC</p>
<hr>
</section>
<section id="seandavicuratedmetagenomicsnextflow-2" class="level3">
<h3 class="anchored" data-anchor-id="seandavicuratedmetagenomicsnextflow-2"><a href="https://github.com/seandavi/curatedMetagenomicsNextflow/commit/d88a87fbfb75f2ecd03ae8ec364ab0c27a346761"><code>seandavi/curatedMetagenomicsNextflow</code></a></h3>
<p><strong>Message:</strong> adds single-sample mode and kneaddata fixes</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-06-03 15:07:29 UTC</p>
<hr>
</section>
<section id="seandavicuratedmetagenomicsnextflow-3" class="level3">
<h3 class="anchored" data-anchor-id="seandavicuratedmetagenomicsnextflow-3"><a href="https://github.com/seandavi/curatedMetagenomicsNextflow/commit/7434d16d55d804140d7891011191845d4fd85128"><code>seandavi/curatedMetagenomicsNextflow</code></a></h3>
<p><strong>Message:</strong> use newest container with metaphlan4.1.0</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-06-03 14:11:06 UTC</p>
<hr>
</section>
<section id="seandavicuratedmetagenomicsnextflow-4" class="level3">
<h3 class="anchored" data-anchor-id="seandavicuratedmetagenomicsnextflow-4"><a href="https://github.com/seandavi/curatedMetagenomicsNextflow/commit/50d3f5bcb98264144bb7f322686cfee7116c7955"><code>seandavi/curatedMetagenomicsNextflow</code></a></h3>
<p><strong>Message:</strong> Merge pull request #25 from ASAP-MAC/main</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-06-03 14:04:30 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbr" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbr"><a href="https://github.com/waldronlab/bugsigdbr/commit/864c8841684579973e5648082433ba0cd4b22f25"><code>waldronlab/bugsigdbr</code></a></h3>
<p><strong>Message:</strong> fix ontology test</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2025-05-29 16:59:55 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdata-2"><a href="https://github.com/waldronlab/curatedMetagenomicData/commit/e77007c870ee42864dc6ab80e4eb54008eb0d303"><code>waldronlab/curatedMetagenomicData</code></a></h3>
<p><strong>Message:</strong> version bump</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-05-23 01:56:36 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdata-3"><a href="https://github.com/waldronlab/curatedMetagenomicData/commit/5aa172ab2a91c45f52f9cc5a394d342eab960b29"><code>waldronlab/curatedMetagenomicData</code></a></h3>
<p><strong>Message:</strong> add authors who updated the vignette</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-05-23 01:56:20 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdata-4"><a href="https://github.com/waldronlab/curatedMetagenomicData/commit/6613f06ffb88e3236e6c1ff5532120809ddae8b9"><code>waldronlab/curatedMetagenomicData</code></a></h3>
<p><strong>Message:</strong> updated vignette</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2025-05-22 19:34:39 UTC</p>
<hr>
</section>
<section id="waldronlabhmp16sdata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhmp16sdata"><a href="https://github.com/waldronlab/HMP16SData/commit/95b7a6d2005d3561f65af583ed4b70813ec4b8a4"><code>waldronlab/HMP16SData</code></a></h3>
<p><strong>Message:</strong> version bump 1.29.1</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 21:58:46 UTC</p>
<hr>
</section>
<section id="waldronlabhmp16sdata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhmp16sdata-1"><a href="https://github.com/waldronlab/HMP16SData/commit/36dfc67b244cef6e510df62fb9a4c5b71fc494a2"><code>waldronlab/HMP16SData</code></a></h3>
<p><strong>Message:</strong> add _pkgdown.yml and .github/workflows/pr_check.yml GHA</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-05-05 21:56:34 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigsimple-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigsimple-2"><a href="https://github.com/waldronlab/bugSigSimple/commit/c06a6067916762c228e0f0f541d7e34322749a55"><code>waldronlab/bugSigSimple</code></a></h3>
<p><strong>Message:</strong> Merge pull request #26 from waldronlab/16-update-student-vignettes-with-modern-analyses</p>
<p><strong>Author:</strong> Chloe Mirzayi</p>
<p><strong>Date:</strong> 2025-05-05 18:54:00 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbr-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbr-1"><a href="https://github.com/waldronlab/bugsigdbr/commit/905b02bd97c3a4265bf7e502ef7fa4bc8791ecc8"><code>waldronlab/bugsigdbr</code></a></h3>
<p><strong>Message:</strong> Bump version to 1.15.2</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2025-04-24 05:07:24 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbr-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbr-2"><a href="https://github.com/waldronlab/bugsigdbr/commit/39fb50faa4c1b8cca222d5b99bbacb3bf50507cd"><code>waldronlab/bugsigdbr</code></a></h3>
<p><strong>Message:</strong> Update documentation</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2025-04-24 05:07:03 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbr-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbr-3"><a href="https://github.com/waldronlab/bugsigdbr/commit/07326cbbe693c95c8f76fd35cebda028d3494233"><code>waldronlab/bugsigdbr</code></a></h3>
<p><strong>Message:</strong> Bump version to 1.15.1</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2025-04-23 23:34:36 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbr-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbr-4"><a href="https://github.com/waldronlab/bugsigdbr/commit/604a171ab5f6d7dd8d8aa4fc34ce2e64a45b3c80"><code>waldronlab/bugsigdbr</code></a></h3>
<p><strong>Message:</strong> Update default to https://doi.org/10.5281/zenodo.15272273</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2025-04-23 23:34:15 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbstats-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbstats-2"><a href="https://github.com/waldronlab/BugSigDBStats/commit/544d8469d41c49d4d8571c87fce87daf49313d35"><code>waldronlab/BugSigDBStats</code></a></h3>
<p><strong>Message:</strong> Turn off publication year until I can figure out how to make it work consistently.</p>
<p><strong>Author:</strong> cmirzayi</p>
<p><strong>Date:</strong> 2025-04-23 15:21:31 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbstats-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbstats-3"><a href="https://github.com/waldronlab/BugSigDBStats/commit/4abb834a3382a123790e294543d13897d41fb865"><code>waldronlab/BugSigDBStats</code></a></h3>
<p><strong>Message:</strong> Update pkgdown.yaml to use 3.21</p>
<p><strong>Author:</strong> Chloe Mirzayi</p>
<p><strong>Date:</strong> 2025-04-23 15:12:18 UTC</p>
<hr>
</section>
<section id="waldronlablefser" class="level3">
<h3 class="anchored" data-anchor-id="waldronlablefser"><a href="https://github.com/waldronlab/lefser/commit/0ac110dc526796d7b609532722b0e14452c0b838"><code>waldronlab/lefser</code></a></h3>
<p><strong>Message:</strong> version bump 1.19.1</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-04-22 20:33:11 UTC</p>
<hr>
</section>
<section id="waldronlablefser-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlablefser-1"><a href="https://github.com/waldronlab/lefser/commit/e3fd48ede0d45f94405982de16965ed39c84d5e0"><code>waldronlab/lefser</code></a></h3>
<p><strong>Message:</strong> update NEWS.md</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-04-22 20:33:03 UTC</p>
<hr>
</section>
<section id="waldronlablefser-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlablefser-2"><a href="https://github.com/waldronlab/lefser/commit/57ec0f8606b1e446d7d79c6b86cc8f7b01e7591a"><code>waldronlab/lefser</code></a></h3>
<p><strong>Message:</strong> document ellipsis in lefserClades</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-04-22 20:32:54 UTC</p>
<hr>
</section>
<section id="waldronlablefser-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlablefser-3"><a href="https://github.com/waldronlab/lefser/commit/07d2eb7f12eb4c2d1c37ae087a2a13d62f9f63f6"><code>waldronlab/lefser</code></a></h3>
<p><strong>Message:</strong> update R dependency to resolve build warning</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-04-22 19:56:38 UTC</p>
<hr>
</section>
<section id="waldronlablefser-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlablefser-4"><a href="https://github.com/waldronlab/lefser/commit/4f5d4ea53c082cb953213b3d7102216033c10d87"><code>waldronlab/lefser</code></a></h3>
<p><strong>Message:</strong> remove blockCol and groupCol from lefser</p>
<p><strong>Author:</strong> LiNk-NY</p>
<p><strong>Date:</strong> 2025-04-22 19:55:25 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataanalyses" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataanalyses"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataAnalyses/commit/0ddc5bef332dbde6c3e9f5442722a5df8c3efd40"><code>waldronlab/MicrobiomeBenchmarkDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Merge pull request #8 from cmirzayi/main</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2025-04-18 22:45:23 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataanalyses-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataanalyses-1"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataAnalyses/commit/1b2b32a25e94fa6949c2000b48e92b10f47a824b"><code>waldronlab/MicrobiomeBenchmarkDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘main’ of https://github.com/cmirzayi/MicrobiomeBenchmarkDataAnalyses</p>
<p><strong>Author:</strong> cmirzayi</p>
<p><strong>Date:</strong> 2025-04-18 21:49:00 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataanalyses-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataanalyses-2"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataAnalyses/commit/ca61d6bf24b10e00ff1ffb8926b292b8cbfe5f55"><code>waldronlab/MicrobiomeBenchmarkDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Add enrichmentSummary HTML datatable</p>
<p><strong>Author:</strong> cmirzayi</p>
<p><strong>Date:</strong> 2025-04-18 21:48:58 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataanalyses-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataanalyses-3"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataAnalyses/commit/cdd29fd88c173a8e7df125eee42c2edea92401a4"><code>waldronlab/MicrobiomeBenchmarkDataAnalyses</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘waldronlab:main’ into main</p>
<p><strong>Author:</strong> Chloe Mirzayi</p>
<p><strong>Date:</strong> 2025-04-17 18:31:43 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataanalyses-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataanalyses-4"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataAnalyses/commit/6348fcdef4feb07ab0416b6f6b132c5cc1a0abab"><code>waldronlab/MicrobiomeBenchmarkDataAnalyses</code></a></h3>
<p><strong>Message:</strong> fix legends to not be cut off</p>
<p><strong>Author:</strong> cmirzayi</p>
<p><strong>Date:</strong> 2025-04-17 18:31:21 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzz" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzz"><a href="https://github.com/waldronlab/bugphyzz/commit/a40066658544eb812729fb4f0bbcc29efda4ae48"><code>waldronlab/bugphyzz</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-04-15 14:24:32 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzz-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzz-1"><a href="https://github.com/waldronlab/bugphyzz/commit/6e1b2a3fdb452ad8868f04b6d07612f1bb0506d7"><code>waldronlab/bugphyzz</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-04-15 14:24:32 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdata" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdata"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/commit/53e4667573c30c257f1828f3af4b252d5c0bb1a1"><code>waldronlab/MicrobiomeBenchmarkData</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-04-15 14:23:01 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdata-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdata-1"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/commit/615e0573eaed0918fd2fa75ef5921556243ae0c7"><code>waldronlab/MicrobiomeBenchmarkData</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-04-15 14:23:01 UTC</p>
<hr>
</section>
<section id="waldronlabhmp16sdata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhmp16sdata-2"><a href="https://github.com/waldronlab/HMP16SData/commit/af66a1c42a2fd0896f9748ba2b7ac6644b9cb732"><code>waldronlab/HMP16SData</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-04-15 14:10:08 UTC</p>
<hr>
</section>
<section id="waldronlabhmp16sdata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhmp16sdata-3"><a href="https://github.com/waldronlab/HMP16SData/commit/a58e99bb5263e78dc9a3c0bb437fbfe548b542ad"><code>waldronlab/HMP16SData</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_21 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2025-04-15 14:10:08 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbcuration" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbcuration"><a href="https://github.com/waldronlab/BugSigDBcuration/commit/c660273db16bc6621a2516d3a0ce8c7b1adc1996"><code>waldronlab/BugSigDBcuration</code></a></h3>
<p><strong>Message:</strong> Merge pull request #691 from waldronlab/fix-no-pmid</p>
<p><strong>Author:</strong> jwokaty</p>
<p><strong>Date:</strong> 2025-03-27 19:32:54 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbcuration-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbcuration-1"><a href="https://github.com/waldronlab/BugSigDBcuration/commit/a76d7d323a2e5f5986dce742db0602a5395cfa8c"><code>waldronlab/BugSigDBcuration</code></a></h3>
<p><strong>Message:</strong> If PMID not found, set as ‘NA’</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2025-03-27 19:25:52 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbcuration-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbcuration-2"><a href="https://github.com/waldronlab/BugSigDBcuration/commit/02f5d3b205ca888817a6e91554657a145d765749"><code>waldronlab/BugSigDBcuration</code></a></h3>
<p><strong>Message:</strong> Fix no PMID, doi link</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2025-03-27 19:17:16 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigsimple-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigsimple-3"><a href="https://github.com/waldronlab/bugSigSimple/commit/70828b7a0519373da79ccdfacac57272cbb9fa04"><code>waldronlab/bugSigSimple</code></a></h3>
<p><strong>Message:</strong> Removed vignettes with older names</p>
<p><strong>Author:</strong> Giacomo Antonello</p>
<p><strong>Date:</strong> 2025-03-27 18:23:13 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigsimple-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigsimple-4"><a href="https://github.com/waldronlab/bugSigSimple/commit/719c0dd0002db8228122aaedca42b6daa74256cd"><code>waldronlab/bugSigSimple</code></a></h3>
<p><strong>Message:</strong> re-roxigenized and re-documented. R CMD check exited with 4 notes, no errors and no warnings</p>
<p><strong>Author:</strong> Giacomo Antonello</p>
<p><strong>Date:</strong> 2025-03-27 14:05:12 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbcuration-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbcuration-3"><a href="https://github.com/waldronlab/BugSigDBcuration/commit/3171dc9fa9070fb9281753dd9a81ed4a30643753"><code>waldronlab/BugSigDBcuration</code></a></h3>
<p><strong>Message:</strong> Merge pull request #686 from waldronlab/strip-html</p>
<p><strong>Author:</strong> jwokaty</p>
<p><strong>Date:</strong> 2025-03-25 20:09:01 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbcuration-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbcuration-4"><a href="https://github.com/waldronlab/BugSigDBcuration/commit/ae699eca7b2271f664a53ee8a38d42c9e008c994"><code>waldronlab/BugSigDBcuration</code></a></h3>
<p><strong>Message:</strong> Squish multiple spaces</p>
<p><strong>Author:</strong> J Wokaty</p>
<p><strong>Date:</strong> 2025-03-25 19:59:42 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-2" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-2"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/e48c5a49175c30130d0bea43efc6ff6fbac3c85a"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> Update body_site_details</p>
<p><strong>Author:</strong> krlong68</p>
<p><strong>Date:</strong> 2025-03-18 20:02:13 UTC</p>
<hr>
</section>
<section id="seandavinextflow_telemetry" class="level3">
<h3 class="anchored" data-anchor-id="seandavinextflow_telemetry"><a href="https://github.com/seandavi/nextflow_telemetry/commit/f8962a57d31ec7b318e1af2be523f09810edb325"><code>seandavi/nextflow_telemetry</code></a></h3>
<p><strong>Message:</strong> Merge pull request #2 from IfeanyiSam/feature/containerization_optimization</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2025-03-13 18:26:38 UTC</p>
<hr>
</section>
<section id="seandavinextflow_telemetry-1" class="level3">
<h3 class="anchored" data-anchor-id="seandavinextflow_telemetry-1"><a href="https://github.com/seandavi/nextflow_telemetry/commit/013641c26c76e90e85e9abbe0a5c5181cfe031bd"><code>seandavi/nextflow_telemetry</code></a></h3>
<p><strong>Message:</strong> feature: Nextflow API Process Optimization</p>
<p><strong>Author:</strong> IfeanyiSam</p>
<p><strong>Date:</strong> 2025-03-09 19:44:07 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-3" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-3"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/96746a3d403dc846eb9794db0c3b9069b5d13931"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> Add dictionaries for “ancestry_details”, “body_site_details”, and “disease_details”; add Svetlana to curator list</p>
<p><strong>Author:</strong> krlong68</p>
<p><strong>Date:</strong> 2025-02-18 19:09:29 UTC</p>
<hr>
</section>
<section id="shbriefcuratedmetagenomicdataodm-4" class="level3">
<h3 class="anchored" data-anchor-id="shbriefcuratedmetagenomicdataodm-4"><a href="https://github.com/shbrief/curatedMetagenomicDataODM/commit/c09d3eee3791a9319a001da8b87b5e7c6a430c7e"><code>shbrief/curatedMetagenomicDataODM</code></a></h3>
<p><strong>Message:</strong> Add non-NCIT terms to “treatment” dictionary; create .owl files for “probing_pocket_depth”, “smoker”, and “feces_phenotype” terms</p>
<p><strong>Author:</strong> krlong68</p>
<p><strong>Date:</strong> 2025-02-07 13:12:20 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbstats-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbstats-4"><a href="https://github.com/waldronlab/BugSigDBStats/commit/0031a5d3cd1da9016daa2433432d6d57bc2272fd"><code>waldronlab/BugSigDBStats</code></a></h3>
<p><strong>Message:</strong> fixed sidebar and updated theme version</p>
<p><strong>Author:</strong> cmirzayi</p>
<p><strong>Date:</strong> 2025-02-06 17:05:43 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbenrich" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbenrich"><a href="https://github.com/waldronlab/BugSigDBEnrich/commit/4312fc46fc13daf08fe341d7fb9e023f05246548"><code>waldronlab/BugSigDBEnrich</code></a></h3>
<p><strong>Message:</strong> Add BM25 column to BugSigDBEnrich</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2025-01-21 04:34:34 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbenrich-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbenrich-1"><a href="https://github.com/waldronlab/BugSigDBEnrich/commit/76a594a127bf3603ff14cff0232cfcbaf38573c1"><code>waldronlab/BugSigDBEnrich</code></a></h3>
<p><strong>Message:</strong> Add script with BM25</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2025-01-09 22:49:36 UTC</p>
<hr>
</section>
<section id="waldronlabtaxppro-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxppro-1"><a href="https://github.com/waldronlab/taxPPro/commit/ebaac7f5bb28462d5386f91965791a0c030edb11"><code>waldronlab/taxPPro</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘main’ of github.com:waldronlab/taxPPro</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2025-01-06 19:39:37 UTC</p>
<hr>
</section>
<section id="waldronlabtaxppro-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxppro-2"><a href="https://github.com/waldronlab/taxPPro/commit/51b27dc2e3066499d377019d54ccb07de1af8b1e"><code>waldronlab/taxPPro</code></a></h3>
<p><strong>Message:</strong> Update pairwisedistances script</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2025-01-06 19:39:17 UTC</p>
<hr>
</section>
<section id="waldronlabcmdmeta" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcmdmeta"><a href="https://github.com/waldronlab/cMDMeta/commit/ca11848d0b38bcfe2947f593f565881b5243d26d"><code>waldronlab/cMDMeta</code></a></h3>
<p><strong>Message:</strong> Clean code</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-12-28 00:31:57 UTC</p>
<hr>
</section>
<section id="waldronlabcmdmeta-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcmdmeta-1"><a href="https://github.com/waldronlab/cMDMeta/commit/b2c6f108d393c1609d56759f3ac8a5a2f6936894"><code>waldronlab/cMDMeta</code></a></h3>
<p><strong>Message:</strong> Add shinydashbordPlus to imports</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-12-27 05:23:23 UTC</p>
<hr>
</section>
<section id="waldronlabcmdmeta-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcmdmeta-2"><a href="https://github.com/waldronlab/cMDMeta/commit/d955f71cd73ac899fd33cc0afd591d949260fe81"><code>waldronlab/cMDMeta</code></a></h3>
<p><strong>Message:</strong> Refactor some of the functions and change function noame to launch the app</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-12-26 22:41:29 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbenrich-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbenrich-2"><a href="https://github.com/waldronlab/BugSigDBEnrich/commit/f78c74919b30ef4672a6ef333d24334a6af835b4"><code>waldronlab/BugSigDBEnrich</code></a></h3>
<p><strong>Message:</strong> Add missing dependency, markdown, and write note when installing locally about having to confirm caching of BugSigDB data</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-12-21 05:01:48 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbenrich-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbenrich-3"><a href="https://github.com/waldronlab/BugSigDBEnrich/commit/865667e34c249c6b1f26aba3bba2feaef61e5cb1"><code>waldronlab/BugSigDBEnrich</code></a></h3>
<p><strong>Message:</strong> Update help for selecting database</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-12-20 22:09:54 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbenrich-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbenrich-4"><a href="https://github.com/waldronlab/BugSigDBEnrich/commit/9453f0f11e3e018af250c04a1f16b1e71507c2eb"><code>waldronlab/BugSigDBEnrich</code></a></h3>
<p><strong>Message:</strong> Change theme, update documentation, bump version, update instructions to try the app, how to run locally, bugphyzz dependency from bioc</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-12-20 21:03:26 UTC</p>
<hr>
</section>
<section id="waldronlabcmdmeta-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcmdmeta-3"><a href="https://github.com/waldronlab/cMDMeta/commit/58e8ff48779d16784810652f6b84c88f5372370f"><code>waldronlab/cMDMeta</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-12-11 02:29:50 UTC</p>
<hr>
</section>
<section id="waldronlabcmdmeta-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcmdmeta-4"><a href="https://github.com/waldronlab/cMDMeta/commit/6de7859b775685d56fdc387c197bbb676940a3df"><code>waldronlab/cMDMeta</code></a></h3>
<p><strong>Message:</strong> Update DESCRIPTION and README file</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-12-11 02:27:26 UTC</p>
<hr>
</section>
<section id="waldronlabtaxppro-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxppro-3"><a href="https://github.com/waldronlab/taxPPro/commit/216f9b3f9d076e0c0d06ae2a207bc4ebf8592b01"><code>waldronlab/taxPPro</code></a></h3>
<p><strong>Message:</strong> Merge pull request #11 from jananiravi/main</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-11-21 18:14:41 UTC</p>
<hr>
</section>
<section id="waldronlabtaxppro-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxppro-4"><a href="https://github.com/waldronlab/taxPPro/commit/97e7eef925237b4009abf9dcad1b8707e1091ad5"><code>waldronlab/taxPPro</code></a></h3>
<p><strong>Message:</strong> Merge pull request #10 from jananiravi/patch-1</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-11-21 18:14:27 UTC</p>
<hr>
</section>
<section id="shbrieflefserbenchmarking" class="level3">
<h3 class="anchored" data-anchor-id="shbrieflefserbenchmarking"><a href="https://github.com/shbrief/lefserBenchmarking/commit/0c83cecf4b06eed75adab7a56a8b967cb43530f9"><code>shbrief/lefserBenchmarking</code></a></h3>
<p><strong>Message:</strong> update README</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-11-19 14:51:28 UTC</p>
<hr>
</section>
<section id="shbrieflefserbenchmarking-1" class="level3">
<h3 class="anchored" data-anchor-id="shbrieflefserbenchmarking-1"><a href="https://github.com/shbrief/lefserBenchmarking/commit/96db3224896e454621e88785f843b9c4f91bf456"><code>shbrief/lefserBenchmarking</code></a></h3>
<p><strong>Message:</strong> minor updates</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-11-19 14:51:15 UTC</p>
<hr>
</section>
<section id="shbrieflefserbenchmarking-2" class="level3">
<h3 class="anchored" data-anchor-id="shbrieflefserbenchmarking-2"><a href="https://github.com/shbrief/lefserBenchmarking/commit/705befb99bdfde2d500ba267fd23acb822003c28"><code>shbrief/lefserBenchmarking</code></a></h3>
<p><strong>Message:</strong> Supplementary Figure 2</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-11-19 14:49:40 UTC</p>
<hr>
</section>
<section id="shbrieflefserbenchmarking-3" class="level3">
<h3 class="anchored" data-anchor-id="shbrieflefserbenchmarking-3"><a href="https://github.com/shbrief/lefserBenchmarking/commit/3c49f01b177540b6ab36e99357636abfffd46c16"><code>shbrief/lefserBenchmarking</code></a></h3>
<p><strong>Message:</strong> updated figures</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-11-19 14:44:20 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzz-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzz-2"><a href="https://github.com/waldronlab/bugphyzz/commit/3af6a514dd1cb1318216a3e0473d31f4ec9182f6"><code>waldronlab/bugphyzz</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_20 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2024-10-29 13:55:47 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzz-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzz-3"><a href="https://github.com/waldronlab/bugphyzz/commit/50358b07950e5ec37527e028cf0af22c1156c6c6"><code>waldronlab/bugphyzz</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_20 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2024-10-29 13:55:47 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdata-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdata-2"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/commit/d9e18a4e0dfe49146f345c5459ce5606320e43f0"><code>waldronlab/MicrobiomeBenchmarkData</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_20 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2024-10-29 13:54:21 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdata-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdata-3"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/commit/d9e041552efc49ad7c06b502c75c16431c408045"><code>waldronlab/MicrobiomeBenchmarkData</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to even y prior to creation of RELEASE_3_20 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2024-10-29 13:54:21 UTC</p>
<hr>
</section>
<section id="waldronlabhmp16sdata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabhmp16sdata-4"><a href="https://github.com/waldronlab/HMP16SData/commit/60c5af944022f955ae8ca96eb9d0bc95746b2897"><code>waldronlab/HMP16SData</code></a></h3>
<p><strong>Message:</strong> bump x.y.z version to odd y following creation of RELEASE_3_20 branch</p>
<p><strong>Author:</strong> lshep</p>
<p><strong>Date:</strong> 2024-10-29 13:47:21 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzz-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzz-4"><a href="https://github.com/waldronlab/bugphyzz/commit/6b04e5f412e60c20771382bce9f5deb19252b5fd"><code>waldronlab/bugphyzz</code></a></h3>
<p><strong>Message:</strong> Update DESCRIPTION - remove taxize - bumbp version</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-10-25 18:43:27 UTC</p>
<hr>
</section>
<section id="shbrieflefserbenchmarking-4" class="level3">
<h3 class="anchored" data-anchor-id="shbrieflefserbenchmarking-4"><a href="https://github.com/shbrief/lefserBenchmarking/commit/9a7a292dc474aed46f00084f1e5f2dd567305dca"><code>shbrief/lefserBenchmarking</code></a></h3>
<p><strong>Message:</strong> minor update in README</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-10-06 03:09:00 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdatalefse" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdatalefse"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataLefse/commit/181c91ec166861a73d229f25419e5bce3fad2632"><code>waldronlab/MicrobiomeBenchmarkDataLefse</code></a></h3>
<p><strong>Message:</strong> gingival data analysis for Fig2</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-09-25 15:49:28 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdatalefse-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdatalefse-1"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataLefse/commit/bc4bea2e4b30dcd3a9de99c8d9696c438af85921"><code>waldronlab/MicrobiomeBenchmarkDataLefse</code></a></h3>
<p><strong>Message:</strong> formatting update for the plots</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-09-25 15:48:41 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdatalefse-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdatalefse-2"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataLefse/commit/a38f5099f278a2b568299c3f8f6f282583e5eeec"><code>waldronlab/MicrobiomeBenchmarkDataLefse</code></a></h3>
<p><strong>Message:</strong> update plot labels</p>
<p><strong>Author:</strong> Sehyun Oh</p>
<p><strong>Date:</strong> 2024-09-20 18:43:20 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdatalefse-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdatalefse-3"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataLefse/commit/a52209717ef689e51e4c7a3ed50a6721cf1a9841"><code>waldronlab/MicrobiomeBenchmarkDataLefse</code></a></h3>
<p><strong>Message:</strong> Add plots comparing lefses</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-09-18 22:16:42 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdatalefse-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdatalefse-4"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataLefse/commit/258a7bfcecd310a732bff197f13aa1bf5cfbd60c"><code>waldronlab/MicrobiomeBenchmarkDataLefse</code></a></h3>
<p><strong>Message:</strong> Add code for comparing lefse results</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-09-18 15:38:11 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdata-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdata-4"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/commit/723cb97bb507b22c3737e5685c72a84d07e96918"><code>waldronlab/MicrobiomeBenchmarkData</code></a></h3>
<p><strong>Message:</strong> Merge pull request #12 from waldronlab/sdgamboa/scml</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-09-02 23:13:45 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataprep" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataprep"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataPrep/commit/3389de0ca72d7b950292deb34f7f2ffab4f75c0f"><code>waldronlab/MicrobiomeBenchmarkDataPrep</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-09-02 17:42:07 UTC</p>
<hr>
</section>
<section id="waldronlabncbitaxon" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabncbitaxon"><a href="https://github.com/waldronlab/ncbitaxon/commit/7df8d8e65a914e1b3c4022e9ba9409e6149a0041"><code>waldronlab/ncbitaxon</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-08-21 15:11:13 UTC</p>
<hr>
</section>
<section id="waldronlabncbitaxon-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabncbitaxon-1"><a href="https://github.com/waldronlab/ncbitaxon/commit/dd30b7fb0c56b7d2898ea692672efd518b0d6b0b"><code>waldronlab/ncbitaxon</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-08-21 15:10:56 UTC</p>
<hr>
</section>
<section id="waldronlabncbitaxon-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabncbitaxon-2"><a href="https://github.com/waldronlab/ncbitaxon/commit/eb722979c1a8b19d8a6dba839e6b098b7b2fff92"><code>waldronlab/ncbitaxon</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-08-21 15:10:02 UTC</p>
<hr>
</section>
<section id="waldronlabncbitaxon-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabncbitaxon-3"><a href="https://github.com/waldronlab/ncbitaxon/commit/bcecde73ddbc56f13067ddc244ddd4e746f8bec6"><code>waldronlab/ncbitaxon</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2024-08-21 15:08:13 UTC</p>
<hr>
</section>
<section id="waldronlabncbitaxon-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabncbitaxon-4"><a href="https://github.com/waldronlab/ncbitaxon/commit/901082455fb1b697d3861b7de17a4ed5e4c466ec"><code>waldronlab/ncbitaxon</code></a></h3>
<p><strong>Message:</strong> Update README</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-08-20 19:54:21 UTC</p>
<hr>
</section>
<section id="waldronlabtaxpprovalidation" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxpprovalidation"><a href="https://github.com/waldronlab/taxPProValidation/commit/6ba9559146e0808a70f3ec152c0968451e31b8e7"><code>waldronlab/taxPProValidation</code></a></h3>
<p><strong>Message:</strong> update README - multistate</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-07-05 20:23:39 UTC</p>
<hr>
</section>
<section id="waldronlabtaxpprovalidation-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxpprovalidation-1"><a href="https://github.com/waldronlab/taxPProValidation/commit/c54e059f4d085f334928304ef142514166f085a2"><code>waldronlab/taxPProValidation</code></a></h3>
<p><strong>Message:</strong> update README</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-07-05 20:22:57 UTC</p>
<hr>
</section>
<section id="waldronlabtaxpprovalidation-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxpprovalidation-2"><a href="https://github.com/waldronlab/taxPProValidation/commit/893f394279c070ac368011cbd193d4f06408d2c5"><code>waldronlab/taxPProValidation</code></a></h3>
<p><strong>Message:</strong> Merge branch ‘main’ of github.com:waldronlab/taxPProValidation</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-07-05 20:19:23 UTC</p>
<hr>
</section>
<section id="waldronlabtaxpprovalidation-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxpprovalidation-3"><a href="https://github.com/waldronlab/taxPProValidation/commit/cdb48ed9d0e331fff2c39b11934a5f8ce5066d80"><code>waldronlab/taxPProValidation</code></a></h3>
<p><strong>Message:</strong> Update README - binary</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-07-05 20:19:03 UTC</p>
<hr>
</section>
<section id="waldronlabtaxpprovalidation-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabtaxpprovalidation-4"><a href="https://github.com/waldronlab/taxPProValidation/commit/7c52cf27c990d86d243d9040bf1f6aece15a8c50"><code>waldronlab/taxPProValidation</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel D. Gamboa-Tuz</p>
<p><strong>Date:</strong> 2024-07-05 20:18:39 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzanalyses" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzanalyses"><a href="https://github.com/waldronlab/bugphyzzAnalyses/commit/05c37dd004e5abea6aafe9c2c564f17af5613cec"><code>waldronlab/bugphyzzAnalyses</code></a></h3>
<p><strong>Message:</strong> update arguments and parameters in the makeSignatures calls</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-06-28 20:31:53 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzanalyses-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzanalyses-1"><a href="https://github.com/waldronlab/bugphyzzAnalyses/commit/d8f8cf29dbe6d974b192332e06a88a8144b03cfd"><code>waldronlab/bugphyzzAnalyses</code></a></h3>
<p><strong>Message:</strong> Add table of ontology.</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-06-11 02:58:44 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzanalyses-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzanalyses-2"><a href="https://github.com/waldronlab/bugphyzzAnalyses/commit/558397c3a331fbda99e5ab60252c243c0c7c1e6f"><code>waldronlab/bugphyzzAnalyses</code></a></h3>
<p><strong>Message:</strong> Update figure dimensions</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-06-07 19:22:56 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzanalyses-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzanalyses-3"><a href="https://github.com/waldronlab/bugphyzzAnalyses/commit/658922eb39b12ff28cd3924bcf886e172b0a5d6f"><code>waldronlab/bugphyzzAnalyses</code></a></h3>
<p><strong>Message:</strong> Add symbols for significant levels to</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-06-07 04:12:10 UTC</p>
<hr>
</section>
<section id="waldronlabbugphyzzanalyses-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugphyzzanalyses-4"><a href="https://github.com/waldronlab/bugphyzzAnalyses/commit/ac215b65fa017d68b454768637b21af8522bc095"><code>waldronlab/bugphyzzAnalyses</code></a></h3>
<p><strong>Message:</strong> Add summary table for dbBact with P-value &lt; 0.1.</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2024-06-07 04:05:00 UTC</p>
<hr>
</section>
<section id="seandavinextflow_telemetry-2" class="level3">
<h3 class="anchored" data-anchor-id="seandavinextflow_telemetry-2"><a href="https://github.com/seandavi/nextflow_telemetry/commit/151554d17febf83b36bbe559c65bd3e58c259dde"><code>seandavi/nextflow_telemetry</code></a></h3>
<p><strong>Message:</strong> add config</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2024-02-07 15:11:13 UTC</p>
<hr>
</section>
<section id="seandavinextflow_telemetry-3" class="level3">
<h3 class="anchored" data-anchor-id="seandavinextflow_telemetry-3"><a href="https://github.com/seandavi/nextflow_telemetry/commit/4387c549aa011bdefaf64afa2ceb166b90360fe7"><code>seandavi/nextflow_telemetry</code></a></h3>
<p><strong>Message:</strong> Dockerfile</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2024-02-07 15:10:52 UTC</p>
<hr>
</section>
<section id="seandavinextflow_telemetry-4" class="level3">
<h3 class="anchored" data-anchor-id="seandavinextflow_telemetry-4"><a href="https://github.com/seandavi/nextflow_telemetry/commit/3035409ba2929450448ec9837abd07ec93d2ed60"><code>seandavi/nextflow_telemetry</code></a></h3>
<p><strong>Message:</strong> README</p>
<p><strong>Author:</strong> Sean Davis</p>
<p><strong>Date:</strong> 2024-02-07 15:10:30 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbpaper" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbpaper"><a href="https://github.com/waldronlab/BugSigDBPaper/commit/3b4c917ebeafe2b02a0fa689cfea5ca03a534b6c"><code>waldronlab/BugSigDBPaper</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2023-10-19 15:37:53 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbpaper-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbpaper-1"><a href="https://github.com/waldronlab/BugSigDBPaper/commit/ec5f4a0acbfd356b3f5a3ab97468a8fb75a0999a"><code>waldronlab/BugSigDBPaper</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2023-10-19 15:36:33 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbpaper-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbpaper-2"><a href="https://github.com/waldronlab/BugSigDBPaper/commit/fe7bebdd799ff94cb9531df901821237ded55036"><code>waldronlab/BugSigDBPaper</code></a></h3>
<p><strong>Message:</strong> Merge pull request #1 from waldronlab/lwaldron-patch-1</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2023-10-19 15:32:19 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbpaper-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbpaper-3"><a href="https://github.com/waldronlab/BugSigDBPaper/commit/df7c2b37b3dbe829bcd94f0b41c5c8429e9eb6d6"><code>waldronlab/BugSigDBPaper</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2023-10-19 15:13:33 UTC</p>
<hr>
</section>
<section id="waldronlabbugsigdbpaper-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabbugsigdbpaper-4"><a href="https://github.com/waldronlab/BugSigDBPaper/commit/69e919a5647d98aeef3b0d96872c33f836c865bc"><code>waldronlab/BugSigDBPaper</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> lgeistlinger</p>
<p><strong>Date:</strong> 2023-09-12 21:02:42 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataprep-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataprep-1"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataPrep/commit/e73cad689095b6adee3e2d268fcab7e3d68ec7b9"><code>waldronlab/MicrobiomeBenchmarkDataPrep</code></a></h3>
<p><strong>Message:</strong> Update README.md</p>
<p><strong>Author:</strong> Samuel D. Gamboa-Tuz</p>
<p><strong>Date:</strong> 2022-08-02 05:00:44 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataprep-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataprep-2"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataPrep/commit/d3852a230c00a2462b1c11a93e46f4bd24d60e20"><code>waldronlab/MicrobiomeBenchmarkDataPrep</code></a></h3>
<p><strong>Message:</strong> correct typo healhy –&gt; healthy</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2022-07-07 13:26:47 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataprep-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataprep-3"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataPrep/commit/8d352345ece347ff24ffff48c3bb7b671074291d"><code>waldronlab/MicrobiomeBenchmarkDataPrep</code></a></h3>
<p><strong>Message:</strong> Change library size of Sammler dataset back to raw counts.</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2022-07-06 14:29:21 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomebenchmarkdataprep-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomebenchmarkdataprep-4"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkDataPrep/commit/a712427aa176415eb38ab173e16ae7e589032738"><code>waldronlab/MicrobiomeBenchmarkDataPrep</code></a></h3>
<p><strong>Message:</strong> update script for counts of spike-in bacteria.</p>
<p><strong>Author:</strong> sdgamboa</p>
<p><strong>Date:</strong> 2022-07-06 00:16:38 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatashinyapp" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatashinyapp"><a href="https://github.com/waldronlab/curatedMetagenomicDataShinyApp/commit/de27cab18f5572cd648ab036badbd579c18f9012"><code>waldronlab/curatedMetagenomicDataShinyApp</code></a></h3>
<p><strong>Message:</strong> change repo/pkg name</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2021-11-29 14:33:52 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatashinyapp-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatashinyapp-1"><a href="https://github.com/waldronlab/curatedMetagenomicDataShinyApp/commit/7ae2aa6a9d82dfee04637856977291800e241a70"><code>waldronlab/curatedMetagenomicDataShinyApp</code></a></h3>
<p><strong>Message:</strong> update instructions for get_metadata</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2021-11-15 16:25:14 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatashinyapp-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatashinyapp-2"><a href="https://github.com/waldronlab/curatedMetagenomicDataShinyApp/commit/9c8abeebbd067f211834d9969bf9f291ae45989d"><code>waldronlab/curatedMetagenomicDataShinyApp</code></a></h3>
<p><strong>Message:</strong> add unlist SRRs</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2021-11-15 16:24:51 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatashinyapp-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatashinyapp-3"><a href="https://github.com/waldronlab/curatedMetagenomicDataShinyApp/commit/81cc6d4867a2c8a1a5162daa3b61fd1659015074"><code>waldronlab/curatedMetagenomicDataShinyApp</code></a></h3>
<p><strong>Message:</strong> update get_metadata output</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2021-11-15 14:44:54 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatashinyapp-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatashinyapp-4"><a href="https://github.com/waldronlab/curatedMetagenomicDataShinyApp/commit/cf7897e9dd1b2b1e379d19e9d433ba1cf456d177"><code>waldronlab/curatedMetagenomicDataShinyApp</code></a></h3>
<p><strong>Message:</strong> update documentation</p>
<p><strong>Author:</strong> Samuel Gamboa</p>
<p><strong>Date:</strong> 2021-11-15 04:38:07 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataterminal" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataterminal"><a href="https://github.com/waldronlab/curatedMetagenomicDataTerminal/commit/ec918d811ccaced0f1020609b079af3caed6cc5a"><code>waldronlab/curatedMetagenomicDataTerminal</code></a></h3>
<p><strong>Message:</strong> add windows instructions</p>
<p><strong>Author:</strong> schifferl</p>
<p><strong>Date:</strong> 2021-11-11 15:44:45 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataterminal-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataterminal-1"><a href="https://github.com/waldronlab/curatedMetagenomicDataTerminal/commit/7f9a5aea5108fa4d95ae7e72e14f23764144c2c2"><code>waldronlab/curatedMetagenomicDataTerminal</code></a></h3>
<p><strong>Message:</strong> merge #11 (cmirzayi/main)</p>
<p><strong>Author:</strong> Lucas Schiffer</p>
<p><strong>Date:</strong> 2021-11-11 15:27:59 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataterminal-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataterminal-2"><a href="https://github.com/waldronlab/curatedMetagenomicDataTerminal/commit/dbca82189af77653ad164c40c33519624399dedc"><code>waldronlab/curatedMetagenomicDataTerminal</code></a></h3>
<p><strong>Message:</strong> Add Windows instructions to README.Rmd</p>
<p><strong>Author:</strong> Chloe Mirzayi</p>
<p><strong>Date:</strong> 2021-11-04 21:39:24 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatapipeline" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatapipeline"><a href="https://github.com/waldronlab/curatedMetagenomicDataPipeline/commit/bdc03f235c04a0d476f0ecc33093f166843cf189"><code>waldronlab/curatedMetagenomicDataPipeline</code></a></h3>
<p><strong>Message:</strong> switch to dgCMatrix</p>
<p><strong>Author:</strong> schifferl</p>
<p><strong>Date:</strong> 2021-10-13 18:05:47 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataterminal-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataterminal-3"><a href="https://github.com/waldronlab/curatedMetagenomicDataTerminal/commit/45c4d4e8bc8c623fdd4977db24776a1cb4c927eb"><code>waldronlab/curatedMetagenomicDataTerminal</code></a></h3>
<p><strong>Message:</strong> use tinyurl link</p>
<p><strong>Author:</strong> schifferl</p>
<p><strong>Date:</strong> 2021-10-13 15:57:56 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdataterminal-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdataterminal-4"><a href="https://github.com/waldronlab/curatedMetagenomicDataTerminal/commit/a31d96eddb419231585a12681261cfb4065f0aed"><code>waldronlab/curatedMetagenomicDataTerminal</code></a></h3>
<p><strong>Message:</strong> Merge pull request #9 from waldronlab/schifferl</p>
<p><strong>Author:</strong> Lucas Schiffer</p>
<p><strong>Date:</strong> 2021-10-13 15:44:26 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatapipeline-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatapipeline-1"><a href="https://github.com/waldronlab/curatedMetagenomicDataPipeline/commit/2b5f05b6120ea1275213acd64c7248620ee5099d"><code>waldronlab/curatedMetagenomicDataPipeline</code></a></h3>
<p><strong>Message:</strong> update make_csv function</p>
<p><strong>Author:</strong> schifferl</p>
<p><strong>Date:</strong> 2021-04-09 20:31:18 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatapipeline-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatapipeline-2"><a href="https://github.com/waldronlab/curatedMetagenomicDataPipeline/commit/345757987584f819ca573ef8478e3d4342978434"><code>waldronlab/curatedMetagenomicDataPipeline</code></a></h3>
<p><strong>Message:</strong> update make_csv function</p>
<p><strong>Author:</strong> schifferl</p>
<p><strong>Date:</strong> 2021-04-06 11:21:34 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatapipeline-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatapipeline-3"><a href="https://github.com/waldronlab/curatedMetagenomicDataPipeline/commit/4235c479fffec501f855de935e5e536e39dc0af1"><code>waldronlab/curatedMetagenomicDataPipeline</code></a></h3>
<p><strong>Message:</strong> add make_csv function</p>
<p><strong>Author:</strong> schifferl</p>
<p><strong>Date:</strong> 2021-04-05 20:58:15 UTC</p>
<hr>
</section>
<section id="waldronlabcuratedmetagenomicdatapipeline-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabcuratedmetagenomicdatapipeline-4"><a href="https://github.com/waldronlab/curatedMetagenomicDataPipeline/commit/4b69c4f01b3604dfa39dd03aa0071e7f306ecac9"><code>waldronlab/curatedMetagenomicDataPipeline</code></a></h3>
<p><strong>Message:</strong> add save_rda function</p>
<p><strong>Author:</strong> schifferl</p>
<p><strong>Date:</strong> 2021-03-31 02:11:57 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomeworkshop" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomeworkshop"><a href="https://github.com/waldronlab/MicrobiomeWorkshop/commit/f31fe45ce8788a9e1b9ee473ee1b65457eebc428"><code>waldronlab/MicrobiomeWorkshop</code></a></h3>
<p><strong>Message:</strong> Almost final version</p>
<p><strong>Author:</strong> Ekaterina Smirnova</p>
<p><strong>Date:</strong> 2019-06-20 13:58:05 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomeworkshop-1" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomeworkshop-1"><a href="https://github.com/waldronlab/MicrobiomeWorkshop/commit/ed5af78a924ac03553353b532d2b8f1f770d4fba"><code>waldronlab/MicrobiomeWorkshop</code></a></h3>
<p><strong>Message:</strong> further updates</p>
<p><strong>Author:</strong> Ekaterina Smirnova</p>
<p><strong>Date:</strong> 2019-06-19 21:20:58 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomeworkshop-2" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomeworkshop-2"><a href="https://github.com/waldronlab/MicrobiomeWorkshop/commit/ff612ec153d3363888d353f74064c1f409a3f441"><code>waldronlab/MicrobiomeWorkshop</code></a></h3>
<p><strong>Message:</strong> updated text and figures</p>
<p><strong>Author:</strong> Ekaterina Smirnova</p>
<p><strong>Date:</strong> 2019-06-19 17:03:26 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomeworkshop-3" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomeworkshop-3"><a href="https://github.com/waldronlab/MicrobiomeWorkshop/commit/a7cc1d2636a21a973ca2d65a2b6202c4c9a3e032"><code>waldronlab/MicrobiomeWorkshop</code></a></h3>
<p><strong>Message:</strong> text updates</p>
<p><strong>Author:</strong> Ekaterina Smirnova</p>
<p><strong>Date:</strong> 2019-06-19 14:03:51 UTC</p>
<hr>
</section>
<section id="waldronlabmicrobiomeworkshop-4" class="level3">
<h3 class="anchored" data-anchor-id="waldronlabmicrobiomeworkshop-4"><a href="https://github.com/waldronlab/MicrobiomeWorkshop/commit/336d541980de6913ba6338f0028fa5a553bdb372"><code>waldronlab/MicrobiomeWorkshop</code></a></h3>
<p><strong>Message:</strong> Add Imports, fixes #9</p>
<p><strong>Author:</strong> Levi Waldron</p>
<p><strong>Date:</strong> 2019-06-12 18:50:29 UTC</p>
<hr>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>